src/autoswagger.ts

```diff
@@ -58,15 +60,14 @@ export class AutoSwagger {
 				<script>
 						window.onload = function() {
 							SwaggerUIBundle({
-								url: "` +
-        url +
-        `",
+								url: "` + url + `",
```

Comment:
```suggestion
								url: "${url}",
```

Should be more readable, wdyt?